packages/framework/src/Twig/PriceExtension.php

```diff
@@ -110,7 +113,7 @@ public function getFilters()
     /**
      * @return array
      */
-    public function getFunctions()
+    public function getFunctions(): array
```

Comment:
please add return typehint also to `getFilters()`

Reply:
Awesome commit :tada: 8d4a56ae4f

Reply:
Fixed